src/Kernel32/storebanned/Kernel32.cs

```diff
@@ -157,6 +157,25 @@ public static partial class Kernel32
         [UnmanagedFunctionPointer(CallingConvention.Winapi)]
         public unsafe delegate bool EnumResLangProc(IntPtr hModule, char* lpType, char* lpName, LANGID wLanguage, void* lParam);
 
+        /// <summary>
+        /// Points to a function that notifies the host that a thread has started to execute.
+        /// </summary>
+        /// <param name="lpThreadParameter">A pointer to the code that has started executing</param>
+        /// <returns>
+        /// The return value indicates the success or failure of this function.
+        /// The return value should never be set to STILL_ACTIVE (259), as noted in <see cref="GetExitCodeThread(IntPtr, out int)"/>.
+        /// </returns>
+        /// <remarks>
+        /// The function to which <see cref="THREAD_START_ROUTINE"/> points is a callback
+        /// function and must be implemented by the writer of the hosting application.
+        ///
+        /// Do not declare this callback function with a void return type and cast the function pointer to
+        /// a pointer to <see cref="THREAD_START_ROUTINE"/> when creating the thread.
+        /// Code that does this is common, but it can crash on 64-bit Windows.
+        /// </remarks>
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
+        public delegate int THREAD_START_ROUTINE([In] IntPtr lpThreadParameter);
```

Comment:
consider using `uint` here since it's `DWORD` in the header file. We sometimes use `int` instead but usually when it's likely that the value will need to be used in other .NET APIs that accept `int`. But in this case, it's an exit code which appears should always be unsigned.